src/lib/AccountStorage.js

```diff
@@ -5,6 +5,8 @@ import { Folder } from './Tree'
 import AsyncLock from 'async-lock'
 import * as localForage from 'localforage'
 
+const storage = (window.browser && browser.storage) || (window.chrome && chrome.storage);
```

Comment:
Please use the browser module https://github.com/marcelklehr/floccus/blob/develop/src/lib/browser-api.js ;)

Reply:
an indeed, thanks for the reference